crates/ekore/Cargo.toml

```diff
@@ -22,3 +22,4 @@ num = "0.4.1"
 
 [dev-dependencies]
 float-cmp = "0.9.0"
+GSL = "7.0"
```

Comment:
I'm still not fully convinced this is what we want, but if so then
1. you need to update Cargo.lock - which by the way currently breaks pre-commit
2. change the workflow: you have to install `gsl` inside the action runner, as the rust packages [relies on the C package](https://github.com/NNPDF/eko/actions/runs/13388466311/job/37390482840#step:9:116)

Reply:
This is now working, so I think we can keep it. But feel free to change it.